src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -0,0 +1,187 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.util.Const;
+
+public class CourseEditSectionRow {
+    public String sectionName;
+    public InstructorAttributes instructor;
+    public List<ElementTag> permissionInputGroup2;
+    public List<ElementTag> permissionInputGroup3;
```

Comment:
yeah, group 2 & 3 here are equivalent to the group 2 & 3 of CourseEditInstructorPanel.java. It's weird to me but this is exactly how it was in the original jsp.